src/rules/no-unsafe-values.js

```diff
@@ -13,6 +17,8 @@ export default {
 
 		messages: {
 			unsafeNumber: "Number outside safe range found.",
+			unsafeInteger: "Integer outside safe range found.",
+			subnormal: "Subnormal numbers are outside the safe range.",
```

Comment:
I think we need to give folks a bit more here because I'm guessing a lot of people don't know what "subnormal" means.

```suggestion
			subnormal: "Unexpected subnormal number '{{value}}` found.",
```

Reply:
Agree, almost nobody deals with these enough, which is why I think this particular lint is so valuable.  Perhaps a link to https://en.wikipedia.org/wiki/Subnormal_number as well?

Reply:
We generally don't put links in the error messages, but I think it would be good to add into the rule description. (At some point we'll create a page for each rule, and that would be a good place to go into more detail.)

Reply:
Whoops, I didn't see this until I just sent the update.  Willfix.